denonavr/ssdp.py

```diff
@@ -49,11 +50,13 @@ def ssdp_request(ssdp_st, ssdp_mx=SSDP_MX):
         'HOST: {}:{}'.format(*SSDP_TARGET),
         '', '']).encode('utf-8')
 
-def get_local_ip(sock):
-    ip_list = socket.gethostbyname_ex(socket.gethostname())[2]
-    for ip in ip_list:
-        if ip.startswith("192."):
-            return ip
+def get_local_ip():
```

Comment:
It is not necessarily the case that local addresses start with 192 - they can really start with 10 172 or 192 - we would like to filter out self-assigned, those that start with 169.
Further, this (and the older) code will give me 127.0.0.1 (local address) not an external IP.  I have done some digging and this function works for me to get my external IP (_happens_ to be 192):
```python
import socket
def get_ip():
    s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
    try:
        # doesn't even have to be reachable
        s.connect(('10.255.255.255', 1))
        IP = s.getsockname()[0]
    except:
        IP = '127.0.0.1'
    finally:
        s.close()
    return IP
```
I found this here after a lot of fails with socket.gethostname() https://stackoverflow.com/questions/166506/finding-local-ip-addresses-using-pythons-stdlib

Reply:
If I do 127.0.0.1 it does not work for me, I do not get any discovered devices.
If I run the code of this PR I get an IP of 192.168.1.xxx which is the IP adress of the computer running the code in my local network, then I do discover my receiver.
If I run the code you perposed I get an IP of 10.xx.xx.xx and I do not discover my receiver using that code....
So this will not work (at least not on my system), but apperently my code does not work on your system.

I will rework to use all IP's discovered just like is done in netdisco library.

Reply:
Note that my system is running a VPN for internet connections, but not for local traffic.

Reply:
The new PR schould fix it for you, it works on my system.

Does it now also work on your system?